minimal-cases/issue-3571/3571.pony

```diff
@@ -0,0 +1,18 @@
+class A
+class BadExtract
+  var a: A trn = A
+  var a_alias: A box = a
+
+  new trn create() => None
+
+  fun ref extract_trn(): A trn^ =>
+    a = A
+
+actor Main
+  new create(env: Env) =>
+    let bad = BadExtract
+    let a_ref: A ref = bad.extract_trn()
+    let a_val: A val = (consume val bad).a_alias
+
+    env.out.print((a_ref is a_val).string())
```

Comment:
The minimal cases directory is for bugs that we don't know how to write automated tests for at this time.

If this is a test case that should cause a compiler error, it should go in the gtest suite instead (for example, in `badpony.cc`, or in one of the other similar compile error test files if you find a more appropriate place).

Reply:
Ah, whoops sorry. Misread what you meant by test case

Reply:
Thank you for mentioning badpony.cc